specs/darwin/crash_logs.table

```diff
@@ -0,0 +1,27 @@
+table_name("crash_logs")
+description("Application Crash Logs")
+schema([
+    Column("process", TEXT, "Name of the process that crashed."),
```

Comment:
The PID is included with the Process name, but I don't split it out on it's own. I believe this wouldn't be too difficult to do and would be pretty useful.
`Process:               loginwindow [102]`